packages/amplify-provider-awscloudformation/src/system-config-manager.ts

```diff
@@ -13,7 +13,7 @@ const logger = fileLogger('system-config-manager');
 const credentialsFilePath = pathManager.getAWSCredentialsFilePath();
 const configFilePath = pathManager.getAWSConfigFilePath();
 
-export function setProfile(awsConfig: $TSAny, profileName: string) {
+export function setProfile(awsConfigInfo: $TSAny, profileName: string) {
```

Comment:
It's not apart of this change, but consider changing the type of `awsConfigInfo` to AwsSdkConfig